src/action_server/actions/arm_goal.py

```diff
@@ -19,6 +19,7 @@ def __init__(self):
         self._required_skills = ['arms']
 
     def _configure(self, robot, config):
+        config = config.semantics
```

Comment:
This renaming will confuse you in the future. If this method gets longer, don't see the rename of the variable. Why not use 

```python 
if config.semantics['side'] == 'left'
``` 
below?